src/pages/[platform]/build-a-backend/add-aws-services/analytics/set-up-analytics/index.mdx

```diff
@@ -520,6 +520,14 @@ public class MyAmplifyApp extends Application {
 
 </InlineFilter>
 
+### View Analytics Console
+
+If you have not saved the link from before, you can still reach it from the terminal. Run the following command for opening the console.
+
+```sh
+amplify console analytics
```

Comment:
I don't think we want a gen 1 command from gen 2. Do we need this? Is there a gen 2 equivalent?

Reply:
I'll remove it for now and ask to see if there's a gen 2 equivalent for this.